modules/sndfile/sndfile.c

```diff
@@ -88,9 +92,25 @@ static SNDFILE *openfile(const struct aufilt_prm *prm, bool enc)
 	SNDFILE *sf;
 	int format;
 
+	char *snd_name_opt;
+
+	if (0 == strcmp(snd_name, "curi")) {
+		snd_name_opt = strm->cname;
+	}
+	else if (0 == strcmp(snd_name, "duri")) {
+		snd_name_opt = strm->duri;
+	}
+	else {
+		(void)re_snprintf(snd_name, sizeof(snd_name),
+				  "%s<->%s",
+				  strm->cname,
```

Comment:
Can we simplify this? Would it be ok if we have only one of this three variants and avoid the new config setting. Maybe this would be enough?

```c
	(void)re_snprintf(filename, sizeof(filename),
			  "%s/dump-%s<->%s-%H-%s.wav",
			  stream_cname(strm), stream_peer(strm),
			  timestamp_print, tm, enc ? "enc" : "dec");
```

You see also: You have to use getter functions for `struct stream`. This have to be added to the API (baresip.h).